xapian-applications/omega/templates/query

```diff
@@ -153,3 +153,4 @@ $map{$cgilist{B},<input type=hidden name=B value="$html{$_}">}
 </body>
 </html>
 $log{query.log}
+$log{search.log,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}	$csv{$query,1}	$hitlist{$id$.}	$topdoc}
```

Comment:
Okay, I've looked up TSV properly now. Either we should use TSV: strip all tab characters from any field they might appear in (in practice this is just the query string), or we should use CSV and `$csv{}`. [The IANA spec for TSV](https://www.iana.org/assignments/media-types/text/tab-separated-values) (which seems to be the only one that actually exists) is clear:

> Note that fields that contain tabs are not allowable in this encoding.

So quoting is unnecessary in TSV, and may cause other problems (like including the quotes in field values when they're read out using compliant libraries; although compliant libraries are really just things like `.split('\x09')`).

(Technically we aren't really using `text/tab-separated-values` because there's no header. CSV can accommodate this in the MIME type, but that's basically irrelevant for us so we shouldn't worry about it.)